recipes/opencv/4.x/conandata.yml

```diff
@@ -40,6 +40,12 @@ patches:
     - patch_file: "patches/4.1.2-0003-find-quirc.patch"
       patch_description: "Robust discovery & injection of quirc"
       patch_type: "conan"
+    - patch_file: "patches/4.5.2-0001-fix-zlib-static-android.patch"
+      patch_description: "Fix discovery of zlib static if Android"
+      patch_type: "conan"
```

Comment:
Maybe portability? Or is the "system" one just named different then the Conan one?